inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/workflow/group/CreateGroupWorkflowDefinition.java

```diff
@@ -85,14 +85,6 @@ public WorkflowProcess defineProcess() {
         initMQResourceTask.addListenerProvider(groupTaskListenerFactory);
         process.addTask(initMQResourceTask);
 
-        // init Sort resource
```

Comment:
We need to adjust the order of creating resources to 1. MQ -> 2. Sink -> 3. Sort -> 4. Source, so as to ensure that after some links fail, data will not be collected or consumed.

Reply:
Understood.